docs/release/release-templates.md

```diff
@@ -0,0 +1,63 @@
+# Release Templates
```

Comment:
Should we refer to this document from release tasks document, specifically from https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#continuously-communicate-key-dates-to-the-community?

Reply:
Sure, linked to this page here: https://github.com/kubernetes-sigs/cluster-api/pull/9572/commits/ebfcf48a381b42727964c4e2fd86729d228f1160#diff-751bae1afb6862debc4d8d6b2d39bba9d2838e720c37c8b2805138c738843025R381